Parse/src/main/java/com/parse/ParsePush.java

```diff
@@ -422,6 +442,14 @@ public void clearExpiration() {
   }
 
   /**
+   * Sets a UNIX epoch timestamp at which this notification should be delivered, in seconds (UTC).
+   * Scheduled time can not be in the past and must be at most two weeks in the future.
+   */
+  public void setPushTime(long time) {
```

Comment:
Oh yes, I didn't notice that. We should use the same time unit (seconds) for all relevant methods.
I think it's okay to add a `Date` version.